src/analytics/lambdas/custom-resource/create-schemas.ts

```diff
@@ -254,6 +254,14 @@ async function createSchemas(props: ResourcePropertiesType, biUsername: string)
   if (sqlStatementsByApp.size == 0) {
     logger.info('Ignore creating schema in Redshift due to there is no application.');
   } else {
+    // const schemasInRedshiftProps: SchemasInRedshiftProps = {
+    //   serverlessRedshiftProps: props.serverlessRedshiftProps,
+    //   provisionedRedshiftProps: props.provisionedRedshiftProps,
+    //   databaseName: props.databaseName,
+    //   dataAPIRole: props.dataAPIRole,
+    // };
+    // await createSchemasInRedshift(sqlStatements, schemasInRedshiftProps);
+
```

Comment:
Is it being used?

Reply:
This part of the code belongs to another module. And merge from the main branch.